BTSPocket/ViewModels/ProfileViewModel.swift

```diff
@@ -22,17 +22,25 @@ struct ProfileViewModel {
                 
                 completition(nil)
             } onError: { error in
-                print(error.localizedDescription)
                 completition(error)
             }
-
         }
     }
     
-    func logout(_ completition: @escaping((_ error: NSError?) -> Void )) {
-        if let token = BTSApi.shared.sessionToken {
+    public func getMemberProfile(_ userId: Int?, _ completition: @escaping( (Result<ProfileData, Error>) -> Void ) ) {
+        if let token = BTSApi.shared.sessionToken,
+           let memberId = userId {
             let headerAuth = ["Authorization": token]
```

Comment:
**spaghetti code:** this token should be  placed o as default header in the **WSAPI** implementation